templates/NOTES.txt

```diff
@@ -24,7 +24,7 @@
 {{- if gt (len $movedList) 0 }}
 Deprecated: The Helm values below should be updated to avoid breaking in a future release!
 {{- if hasKey .Values "cemanager" }}
-  ⚠ The "cemanager" Kubernetes Service will be renamed to "coderd" once moved.
+  ⚠ The "cemanager" Kubernetes Service will be renamed to "coderd".
```

Comment:
Not sure if this is accurate but I thought "once moved" was confusing

Reply:
This triggers if any `cemanager.*` keys exist in the helm. I wasn't sure how to word it, but the service automatically gets renamed to "coderd" once the old keys have been renamed.

Reply:
Maybe `once all cemanager.* Helm values are renamed, the "cemanager" Kubernetes Service will be renamed to "coderd"`?